### PR TITLE
feat(apple): Add docs on Core Data APM

### PR DESCRIPTION
**Pull request by brustolin**
Documentation for Core Data auto transactions experimental feature added by https://github.com/getsentry/sentry-cocoa/pull/1682

### CODE REVIEW

File: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -115,6 +115,38 @@ SentrySDK.start { options in
 [NSString]: https://developer.apple.com/documentation/foundation/nsstring
 [NSBundle]: https://developer.apple.com/documentation/foundation/nsbundle
 
+## Core Data Instrumentation
+
+<Alert level="info" title="Important">
+
+This is an experimental feature and requires an opt-in. Experimental features are still a work-in-progress and may have bugs.
```
Comment (bruno-garcia) [inline src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx:122]:
```suggestion
This is an experimental feature and requires an opt-in. Experimental features are still a work-in-progress and may have bugs. We recognize the irony.
```

File: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 2]
```diff
@@ -115,6 +115,38 @@ SentrySDK.start { options in
...
+
+</Alert>
+
+The Sentry SDK adds spans for Core Data operations to ongoing transactions bound to the scope. Currently, the SDK supports fetch and save operations with [NSManagedObjectContext][NSManagedObjectContext].
```
Comment (bruno-garcia) [inline src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx:126]: Are we reusing these URLs? Isn't it easier to inline them?

```suggestion
The Sentry SDK adds spans for Core Data operations to ongoing transactions bound to the scope. Currently, the SDK supports fetch and save operations with [NSManagedObjectContext](https://developer.apple.com/documentation/coredata/nsmanagedobjectcontext).
```

If not, please add it to the bottom of the page with the rest, but imo that just makes things more complicated to maintain

File: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 3]
```diff
@@ -115,6 +115,38 @@ SentrySDK.start { options in
...
+This feature is experimental and is disabled by default. 
+
```
Comment (bruno-garcia) [inline src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx:128]: We already say this above in the note

```suggestion
```

File: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 4]
```diff
@@ -115,6 +115,36 @@ SentrySDK.start { options in
...
+```objc {tabTitle:Objective-C}
+@import Sentry;
+
+[SentrySDK startWithConfigureOptions:^(SentryOptions *options) {
+    options.dsn = @"___PUBLIC_DSN___";
+    options.enableCoreDataTracking = YES;
+}];
+```
+[NSManagedObjectContext]: https://developer.apple.com/documentation/coredata/nsmanagedobjectcontext
```
Comment (brustolin) [inline src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx:146]:
```suggestion
```

**Comment by bruno-garcia** [comment]
Merge once this is merged/shipped: https://github.com/getsentry/sentry-cocoa/pull/1682/

**Review by philipphofmann — APPROVED** [review]
LGTM 🚀

**Pull request merged**